src/main/java/org/bihealth/mi/easybus/implementations/email/ConnectionIMAPSettings.java

```diff
@@ -123,26 +123,61 @@ public static void checkPort(int object) {
     private boolean                       ssltlsIMAP           = true;
     /** Use ssl/tls (=true) or starttls (=false) for SMTP connection */
     private boolean                       ssltlsSMTP           = true;
-    /** Password accessor */
-    private PasswordProvider              provider;
-
-    /**
-     * Creates a new instance
+    /** Password accessor for receiving */
+    private PasswordProvider              imapProvider;
+    /** E-mail address sending */
+    private String                        smptEmailAddress;
+    /** Password accessor for sending */
```

Comment:
It's weird that the class is called ConnectionIMAPSettings but contains imap-fields as well as smpt-fields. Why? Maybe rename class?

Reply:
Weil es eine Klasse ConnectionIMAP gibt, die ConnectionEmail erweitert. Entsprechend gibt es auch eine Klasse ConnectionIMAPSettings .
Vermutluch heißen diese Klassen so, weil am Anfang auch eine POP3-Unterstützung im Raum stand, für senden aber immer SMTP.